spring-integration-core/src/main/java/org/springframework/integration/endpoint/PollingConsumer.java

```diff
@@ -72,6 +74,26 @@ public void setReceiveTimeout(long receiveTimeout) {
 	}
 
 	@Override
+	public MessageChannel getInputChannel() {
+		return this.inputChannel == null ? null : this.inputChannel;
```

Comment:
Overdid: if `null`, return `null` :smile: 
From other side it can't be `null`. See ctor:
```
Assert.notNull(inputChannel, "inputChannel must not be null");
```